demo/scripts/common.sh

```diff
@@ -52,3 +52,5 @@ SCRIPTS_DIR="$(cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd)"
 # The name of the kind image to build / run
 : ${KIND_IMAGE:="kindest/node:${KIND_K8S_TAG}"}
 
+# Container tool, e.g. docker/podman
+: ${DOCKER:="docker"}
```

Comment:
Not critical for this PR. Is there a more generic term that we can use for `DOCKER`? (may affect more files, but it may be good to do as a follow up)

Reply:
Yes, usually it's `CONTAINER_TOOL` or something along those lines. As long as you're OK with moving away from DOCKER :) Ideally all Dockerfiles and directory names should change as well.

Reply:
I think I would be ok with it, but may require some more discussion on our end if we start renaming things outside of the environment and make variables.

What is the generic standard for `Dockerfile`? From my perspective, this is an implied API and not just a file name.

Reply:
Sounds good.
`Containerfile` is more inclusive :)

Reply:
To be clear, I don't think we should rename the `Dockerfile` to `Containerfile` at this stage. See https://github.com/docker/cli/issues/3251

Reply:
I've replaced `docker` command with `CONTAINER_TOOL` variable.

Reply:
It make the code look better than `if [[ "${DOCKER}" == "podman" ]]` :smile: